man/src/oxide-config.1.md

```diff
@@ -0,0 +1,267 @@
+% OXIDE-CONFIG(1) oxide-config 0.1.0
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-config - config
+
+# DESCRIPTION
+
+Define the behaviour of oxidewm. You can change style, layout, keybindings and more.
+The config file is written using YAML syntax.
+If the home config file is not existing, default values will be used but commands like `exec` and `exec_always` will not be working.
+
+# FILES
+
+When starting, Oxide has two paths it searches for config files.
+
+**~/.config/oxide/config.yml**
+: Home config file
+
+**/etc/oxide/config.yml**
+: System config file
+
+# KEYBINDING
+
+## KEYS
+
+A keybinding has to consist of at least one or more MODIFIERS and exactly one normal key such as 't' for example.
+
+## MODIFIER
+
+**M**
+: Meta key
+
+**A**
+: ALT key
+
+**C**
+: CONTROL key
+
+**S**
+: SHIFT key
+
+# COMMANDS
+
+Commands consist of a command and optional arguments.
+
+## COMMAND
+
+Move **args** [MOVEMENT]
+: Move window
```

Comment:
I know this is very picky but in the lines 52 - 94 the descriptions should be written in lowercase letters like you did in lines 99 and following to be consistent and give it a better look :)